server/index.js

```diff
@@ -13,16 +13,16 @@ app.all('*', function (req, res, next) {
 
 app.get('*', async (req, res) => {
   /** @type {String} */
-  const code = req.query.code;
+  const {code, client_id, client_secret} = req.query;
   try {
     console.log(`Authenticating Code: ${code.substr(0, 3)}...`);
     const response = await axios.post(
       'https://github.com/login/oauth/access_token',
```

Comment:
Just posting our convo earlier here:

* We should allow changing this base URL too (e.g. `github.ibm.com`)
* And if the user *does* provide a base URL, then we should make sure there is no way our `process.env` secrets can ever be sent there.

Reply:
If we start to allow proving a base URL, do we have a responsibility to add rate limiting or something so this can't easily be abused?

e.g. As a malicious user:
1. I send a base URL to Issueist that isn't GitHub and/or that I don't control
2. Issueist is now spamming some rando's server